pypesto/sampling/pymc3.py

```diff
@@ -26,12 +30,12 @@ class Pymc3Sampler(Sampler):
     step_function:
         A pymc3 step function, e.g. NUTS, Slice. If not specified, pymc3
         determines one automatically (preferable).
-    **options:
+    **kwargs:
         Options are directly passed on to `pymc3.sample`.
     """
 
-    def __init__(self, step_function=None, **options):
-        super().__init__(options)
+    def __init__(self, step_function=None, **kwargs):
+        super().__init__(kwargs)
```

Comment:
A bit late, but: not `**kwargs`?

Reply:
the superclass handlers take a dict, not kwargs ... not ideal but works